xbmc/windowing/wayland/WinSystemWaylandEGLContext.cpp

```diff
@@ -28,6 +30,10 @@
 
 using namespace KODI::WINDOWING::WAYLAND;
 
+CWinSystemWaylandEGLContext::CWinSystemWaylandEGLContext()
+: m_eglContext{EGL_PLATFORM_WAYLAND_EXT, "EGL_EXT_platform_wayland"}
```

Comment:
This strict check for `EGL_EXT_platform_wayland` will not work on libmali for Mali 450 gpu, that driver reports EGL version 1.4 but report client extensions `EGL_KHR_platform_wayland` and `EGL_KHR_platform_gbm` from EGL 1.5 spec instead of `EGL_EXT_platform_wayland` and `EGL_MESA_platform_gbm` for EGL 1.4

Reply:
Yeah, then the driver is broken. Anyway does it work with `eglGetDisplay`?